src/esp/bindings/ShortestPathBindings.cpp

```diff
@@ -80,6 +80,9 @@ void initShortestPathBindings(py::module& m) {
            "meters_per_pixel"_a, "height"_a)
       .def("get_random_navigable_point", &PathFinder::getRandomNavigablePoint,
            "max_tries"_a = 10)
+      .def("get_random_navigable_point_near",
+           &PathFinder::getRandomNavigablePointAroundSphere, "circle_center"_a,
+           "radius"_a, "max_tries"_a = 10)
```

Comment:
Maybe a helptext describing what's going on here? :)

Reply:
I don't want this to be that public of a function, yet, but I can add a docstring.